server/lib/models/matchmaking-map-pools.ts

```diff
@@ -8,6 +8,7 @@ interface DbMapPool {
   matchmaking_type: MatchmakingType
   start_date: Date
   maps: string[]
+  max_veto_count: number
```

Comment:
instead of just adding another field here can you please move these types to the standard pattern used everywhere else in the app?